packages/server/api/src/app/flows/flow-run/flow-run-side-effects.ts

```diff
@@ -51,11 +52,13 @@ export const flowRunSideEffects = {
             .onFinish({ projectId: flowRun.projectId, tasks: flowRun.tasks! })
         if (flowRun.environment === RunEnvironment.PRODUCTION) {
             if (isFailedState(flowRun.status)) {
-                await issuesService.add({
+                const issue = await issuesService.add({
                     flowId: flowRun.flowId,
                     projectId: flowRun.projectId,
                     flowRunCreatedAt: flowRun.created,
                 })
+
+                await alertsService.sendAlertOnFinish({ issue, flowRunId: flowRun.id })
```

Comment:
Can we rename this to sendAlertsOnRunFinish ? and rename this method to calculateDelayForPausedRun?